src/components/searchbars/ChipSearchbar.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+    <div class="search-bar">
+        <v-combobox
+            ref="input"
+            label="Search anything..."
+            placeholder="Adresses, dishes, ingredients, food types..."
+            :items="categories"
+            multiple
+            clearable
+            chips
+            solo
+            append-icon=""
+            deletable-chips
+            hide-no-data
+            @change="search"
+        >
+        <template v-slot:item="{item}">
+            <v-chip class="ma-2">
+                {{item}}
+            </v-chip>
+        </template>
+        <template v-slot:no-data>
+        </template>
+        </v-combobox>
+    </div>
+</template>
+
+<script>
+export default {
+    name: 'ChipSearchbar',
+    props: ["categories"],
+    data () {
+        return {
+            isLoading: false,
```

Comment:
Remove comma if you are not adding another attribute after isLoading

Reply:
This is probably an older version. Chipsearchbar doesn't have data anymore.